app/pathogen/arbovirus/dashboard/(visualizations)/country-seroprevalence-comparison-scatter-plot.tsx

```diff
@@ -0,0 +1,204 @@
+import { useContext, useMemo } from "react";
+import { ArboContext } from "@/contexts/arbo-context/arbo-context";
+import { arbovirusesSF } from "./recharts";
+import {
+  CartesianGrid,
+  ErrorBar,
+  Scatter,
+  ScatterChart,
+  Tooltip,
+  XAxis,
+  YAxis,
+  ResponsiveContainer,
+  Label,
+} from "recharts";
+import { groupDataForRecharts } from "./group-data-for-recharts";
+import uniq from "lodash/uniq";
+import { typedObjectKeys } from "@/lib/utils";
+import { ContentType } from "recharts/types/component/Tooltip";
+
+const CustomTooltip: ContentType<string, string> = ({ active, payload, label }) => {
+  if (active && payload && payload.length) {
+    const seroprevalencePayload = payload.find((element) => element.name === 'seroprevalence')?.payload;
+
+    if(!seroprevalencePayload) {
+      return null;
+    }
+
+    const seroprevalence = seroprevalencePayload.seroprevalence;
+    const seroprevalenceCalculated95CILower = seroprevalencePayload.seroprevalenceCalculated95CILower;
+    const seroprevalenceCalculated95CIUpper = seroprevalencePayload.seroprevalenceCalculated95CIUpper;
+
+    return (
+      <div className="bg-white p-4">
+        <p> Seroprevalence: {seroprevalence}% </p>
+        <p> 95% CI: [{seroprevalenceCalculated95CILower}%, {seroprevalenceCalculated95CIUpper}%] </p>
+      </div>
+    );
+  }
+
+  return null;
+};
+
+export const CountrySeroprevalenceComparisonScatterPlot = () => {
+  const state = useContext(ArboContext);
+
+  const stateFilteredDataWithEstimateNumbers = useMemo(
+    () =>
+      state.filteredData
+        .filter(
+          (dataPoint) =>
+            dataPoint.seroprevalenceCalculated95CILower &&
+            dataPoint.seroprevalenceCalculated95CIUpper
```

Comment:
Some studies don't have numerators and sample sizes (?) and don't have these fields because of that.